samples/System.Devices.Gpio.Samples/README.md

```diff
@@ -0,0 +1,3 @@
+# System.Device.Gpio samples have moved!
```

Comment:
Instead of adding new READMEs and keeping the folders around, can you instead add this to a heading to the root README after archived projects?

https://github.com/dotnet/corefxlab#archived-projects

Reply:
It seems odd to have, an essentially empty directory, with the project format. But I don't feel strongly about it.

Reply:
I'll keep them for now, given that I do believe they will be useful, but if I get more feedback saying to remove them, or you feel strongly about removing them, then I'll go ahead and remove.